esp/src/src-react/components/DaliAdmin.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { Pivot, PivotItem } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { pushUrl } from "../util/history";
+import { pivotItemStyle } from "../layouts/pivot";
+import { GetDFSCSV } from "src/WsDali";
+import nlsHPCC from "src/nlsHPCC";
+
+interface DaliAdminProps {
+    filter?: object;
+    tab?: string;
+    name?: string;
+    baseDn?: string;
+}
+
+export const DaliAdmin: React.FunctionComponent<DaliAdminProps> = ({
+    filter,
+    tab = "GetDFSCSV",
+    name,
+    baseDn
+}) => {
+
+    return <>
+    <SizeMe monitorHeight>{({ size }) =>
+        <Pivot overflowBehavior="menu" style={{ height: "100%" }} selectedKey={tab} onLinkClick={evt => pushUrl(`/wsdali/${tab}/${evt.props.itemKey}`)}>
+            <PivotItem headerText={nlsHPCC.GetDFSCSV} style={pivotItemStyle(size)} >
```

Comment:
Each pivot item will need a "key" attribute - which is what the "selectedKey" refers to in the main Pivot